docs/source/getting_started/amd-installation.rst

```diff
@@ -0,0 +1,80 @@
+.. _installation:
+
+Installation with ROCm
+============
+
+vLLM 0.2.x onwards supports model inferencing and serving on AMD GPUs with ROCm.
+At the moment AWQ quantization is not supported in ROCm, but SqueezeLLM quantization has been ported.
+Datatypes currently supported in ROCm are FP16 and BF16.
+
+Requirements
+------------
+
+* OS: Linux
+* Python: 3.8 -- 3.11 (Verified on 3.10)
+* GPU: MI200s
+* Pytorch 2.0.1/2.1.1
+* ROCm >= 5.7.0
+
+
+.. _build_from_source:
+
+Build from source with docker
```

Comment:
Maybe "Option 1: Installation using Docker" ?

Reply:
Thank you. We have updated the documentations. Please take a look.